src/adapters/adapter.js

```diff
@@ -93,4 +100,64 @@ export default class Adapter {
     console.log('getUserIdByUserName not implemented by this adapter.');
     return name;
   }
+
+  getRoleIdByRoleName (name) {
+    console.log('getRoleIdByRoleName not implemented by this adapter');
+    return name;
+  }
+
+  getRolesForUser () {
+    return [];
+  }
+
+  async initUsers() {
+    await this.bot.databaseInitialized();
+
+    this.adapterUsers = this.bot.db.get(`exobot-users.${this.name}`).value();
```

Comment:
how do you feel about making the string `exobot-users` a constant at the top of the file? `const USERS_DB = 'exobot-users'`